datafusion/common/src/dfschema.rs

```diff
@@ -1146,35 +1212,6 @@ mod tests {
         Ok(())
     }
 
-    #[allow(deprecated)]
```

Comment:
Dropped as we have other tests duplicating this behavior, furthermore these tests referred to deprecated and already dropped methods